simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,24 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    lenLst = len(lst)
```

Comment:
Does not look like you ever use this one in your code.

Reply:
fix.